library/rocket.chat

```diff
@@ -1,10 +1,14 @@
-# this file is generated via https://github.com/RocketChat/Docker.Official.Image/blob/fb09fece013b05ce7c0d8a42b646543a75fda57a/generate-stackbrew-library.sh
+# this file is generated via https://github.com/RocketChat/Docker.Official.Image/blob/6bb91d4971eb1d1c8e7fb84ec2d646c2b9474818/generate-stackbrew-library.sh
 
 Maintainers: Rocket.Chat Image Team <buildmaster@rocket.chat> (@RocketChat)
 GitRepo: https://github.com/RocketChat/Docker.Official.Image.git
 
-Tags: 6.5.2, 6.5, 6, latest
-GitCommit: fb09fece013b05ce7c0d8a42b646543a75fda57a
+Tags: 6.6.0, 6.6, 6, latest
+GitCommit: 6bb91d4971eb1d1c8e7fb84ec2d646c2b9474818
+Directory: 6.6
+
+Tags: 6.5.3, 6.5
```

Comment:
Any particular reason to keep these old versions listed here? (It's my understanding that they're now EOL?)

Removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
hey @tianon , I even checked the documentation before submitting the PR https://docs.rocket.chat/customer-center/support-center/premium-support-plans/support-prerequisites-and-version-durability#version-end-of-life

not sure if they bumped the supported versions a bit, but it states 6.3.x EOL is end of this month, so I left it here.

Reply:
Oh interesting - I was looking at https://endoflife.date/rocket-chat but maybe that's outdated (or talking about a different version). :+1: :heart:

(Probably worth sending them a PR if it's outdated though!)